lib/easybill-rest-client/api_client.rb

```diff
@@ -26,55 +29,58 @@ def request_collection(method, endpoint, params = {})
     end
 
     def request(method, endpoint, params = {})
-      Retryable.retryable(
-        tries: tries,
-        sleep: retry_cool_off_time,
-        on: EasybillRestClient::TooManyRequests
-      ) do
-        response = perform_request(method, endpoint, params)
-        process_response(response)
+      @request_logger = RequestLogger.new(logger: logger, request_id: Time.now.to_f)
```

Comment:
@tsujigiri You might also not need all the 32 bytes of `SecureRandom.hex`, and make it faster by using `SecureRandom.hex(3)`.